README.md

```
@@ -17,11 +17,16 @@ K-뷰티 플레이어와 일본인 소비자를 연결하는 bridge 역할을 
     <th> <a href="https://github.com/hamxxn">권동희</a></th>
     </tr>
     <tr>
-    	<td><img src="https://github.com/Jeong-Ag.png" width="360px"></td>
-    	<td><img src="https://github.com/ljh0608.png" width="360px"></td>
-      <td><img src="https://github.com/kojesung.png" width="360px"></td>
-    	<td><img src="https://github.com/heeyyeon.png" width="360px"></td>
-      <td><img src="https://github.com/hamxxn.png" width="360px"></td>
+    	<td><img width="360px" height="1063" alt="정은" src="https://github.com/user-attachments/assets/30f54bac-2170-4c58-8f29-4fca629ac7a2" /></td>
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 5

(MD010, no-hard-tabs)